pkg/instances/instances.go

```diff
@@ -264,42 +264,44 @@ func (i *instances) GetControlPlaneStatus() []StatusOutput {
 				options.LabelSelector = labels.FormatLabels(labelSelector)
 
 				p, err := i.kubeClient.CoreV1().Pods(v1.NamespaceAll).List(options)
-				if err == nil && len(p.Items) == 1 {
-					pod := p.Items[0]
-
-					image := pod.Spec.Containers[0].Image
-					namespace := pod.GetNamespace()
-					age := age.GetAge(pod.CreationTimestamp.Time)
-					created := pod.CreationTimestamp.Format("2006-01-02 15:04.05")
-					version := image[strings.IndexAny(image, ":")+1:]
-					status := ""
-
-					if pod.Status.ContainerStatuses[0].State.Waiting != nil {
-						status = fmt.Sprintf("Waiting (%s)", pod.Status.ContainerStatuses[0].State.Waiting.Reason)
-					} else if pod.Status.ContainerStatuses[0].State.Running != nil {
-						status = "Running"
-					} else if pod.Status.ContainerStatuses[0].State.Terminated != nil {
-						status = "Terminated"
-					}
+				if err == nil {
+					for _, pod := range p.Items {
+						name := pod.Name
```

Comment:
can you add in replicas also for https://github.com/dapr/dashboard/issues/108 ?
`replicas := len(p.Items)`

Reply:
No need, I am listing them all here.